slides/evaluation/slides-evaluation-generr.tex

```diff
@@ -120,7 +120,7 @@
       loss $L$ or metric $\rho$.
   \item Nomenclature: ERM and \textbf{inner loss}; evaluation and \textbf{outer loss}. 
           % In this case, it holds by the law of the large numbers for the empirical risk $\riske$ that
-  % $$\E_{\D\sim(\Pxy)^{\ntrain}}\left[\frac{1}{n}\sumin L\left(\yi, \fhDlam(\xi)\right)  \right] \xrightarrow{n \to \infty} \GE(\inducer, \lamv, \ntrain, \rhoL) .$$
+  % $$\E_{\D\sim(\Pxy)^{\ntrain}}\left[\frac{1}{n}\sumin L\left(\yi, \fhDlam(\xi)\right)  \right] \xrightarrow{n \to \infty} \GE(\in, \lamv, \ntrain, \rhoL) .$$
```

Comment:
should be "\ind"? (Yes, is commented out, but if it's not deleted, it should be correct still)